torchci/lib/bot/autoLabelBot.ts

```diff
@@ -324,56 +354,84 @@ function myBot(app: Probot): void {
     await addNewLabels(labels, labelsToAdd, context);
   });
 
-  app.on(["pull_request.opened", "pull_request.edited"], async (context) => {
-    const labels: string[] = context.payload.pull_request.labels.map(
-      (e) => e["name"]
-    );
-    const owner = context.payload.repository.owner.login;
-    const repo = context.payload.repository.name;
-    const title = context.payload.pull_request.title;
-    const filesChangedRes = await context.octokit.paginate(
-      "GET /repos/{owner}/{repo}/pulls/{pull_number}/files",
-      {
-        owner,
-        repo,
-        pull_number: context.payload.pull_request.number,
-        per_page: 100,
+  app.on(
+    ["pull_request.opened", "pull_request.edited", "pull_request.synchronize"],
```

Comment:
This looks slightly different than https://github.com/pytorch/pytorch/blob/main/.github/workflows/labeler.yml#L10 but having `pull_request.edited` makes sense to me as it covers updating base branch